src/pages/Explorer/Layouts/SocialTrend.svelte

```diff
@@ -0,0 +1,30 @@
+<script>
+    export let item;
+</script>
+
+<div class="row column">
```

Comment:
1. `row` is not needed, when the `column` applied;
2. `column` is not needed, because it is default behaviour of the `div` element